pkg/types/string.go

```diff
@@ -15,6 +15,14 @@ type String struct {
 	sql.NullString
 }
 
+// NewString constructs a new non-NULL String from s.
+func NewString(s string) String {
```

Comment:
I suggested the name `MakeString` because for `NewString`, I'd kind of expect it to return `*String`, similar to `new(String)` or pretty much any other `New*` function I can think of.